internal/users/manager.go

```diff
@@ -413,3 +413,13 @@ func getActiveUsers(procDir string) (activeUsers map[string]struct{}, err error)
 	}
 	return activeUsers, nil
 }
+
+// GenerateID generates an integer number based on the provided string.
+func GenerateID(str string) int {
+	var sum int
```

Comment:
You need to lower the string, so that we always return the same ID from different case of `str` (which I think is general for now).

Also, a couple of tests for this one (at least with different cases), as it’s an exported function will be great in `users/`.

Finally, I think a nss and broker case with a different case will illustrate it well too.